specs/jointcal/hsc_r.yaml

```diff
@@ -0,0 +1,42 @@
+# Specifications for jointcal metrics against testdata_jointcal
+# hsc r-band datasets.
+
+---
+id: 'base'
+provenance_query:
+  # Prototype for query provenance. This needs to be figured out.
+  dataset_repo_url: 'https://github.com/lsst/testdata_jointcal.git'
+
+---
+# jointcal.astrometry_ndof.hsc_11_visits
+name: "hsc_11_visits"
+  base: ["#astrometry_ndof-base", "#base"]
```

Comment:
I think the syntax here will be `'astrometry_ndof.yaml#astrometry_ndof-base'` to help resolve that partial.